docs/hw-platforms.rst

```diff
@@ -0,0 +1,348 @@
+Supported Boards
+================
+
+The following XMC microcontroller boards are supported by XMC for Arduino:
+
+.. list-table:: Supported Microcontroller Boards
+   :header-rows: 1
+
+   * - Picture
+     - Board Name
+     - MCU
+     - Form Factor
+   * - .. image:: img/KIT_XMC11_BOOT_001.jpg
+     - :ref:`KIT_XMC11_BOOT_001`
+     - XMC1100
+     - Arduino Uno
+   * - .. image:: img/KIT_XMC13_BOOT_001.jpg
+     - :ref:`KIT_XMC13_BOOT_001`
+     - XMC1302
+     - Proprietary
+   * - .. image:: img/KIT_XMC14_2GO.png
+     - :ref:`KIT_XMC14_2GO`
+     - XMC1404
+     - Shield2Go
+   * - .. image:: img/KIT_XMC1400_ARDUINO.jpg
+     - :ref:`KIT_XMC1400_ARDUINO`
+     - XMC1402
+     - Arduino Uno
+   * - .. image:: img/KIT_XMC_PLT2GO_XMC4200.jpg
+     - :ref:`KIT_XMC_PLT2GO_XMC4200`
+     - XMC4200
+     - Arduino Uno, Shield2Go, mikroBUS
+   * - .. image:: img/KIT_XMC_PLT2GO_XMC4400.jpg
+     - :ref:`KIT_XMC_PLT2GO_XMC4400`
+     - XMC4400
+     - Arduino Uno, Shield2Go, mikroBUS
+   * - .. image:: img/KIT_XMC47_RELAX_5V_AD_V1.jpg
+     - :ref:`KIT_XMC47_RELAX_5V_AD_V1`
+     - XMC4700
+     - Arduino Uno
+
+.. note::
+    For all Kit 2Go or MS 2Go sensor boards please select ``XMC1100 XMC2Go`` in the Arduino IDE.
+
+.. _KIT_XMC11_BOOT_001:
+
+KIT_XMC11_BOOT_001
+------------------
+
+* Name in Arduino IDE: ``XMC1100 Boot Kit``
+* `Product Page <https://www.infineon.com/cms/de/product/evaluation-boards/kit_xmc11_boot_001/>`__
+
+.. image:: img/KIT_XMC11_BOOT_001.jpg
+    :width: 400
+
+The XMC1100 Boot Kit board consists of a XMC1100 microcontroller with a debugger implemented by a XMC4200 microcontroller.
```

Comment:
Aren´t all these boards features description already available in the product page?

Reply:
Would not the pinout diagram be sufficient as a picture?

Reply:
I changed the description to a very short version for quick reference in the docs. The main part is on the board website.

Reply:
The  2 pics are redundant to me, but fine 👍